core/src/main/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtil.java

```diff
@@ -1078,5 +1078,25 @@ public static boolean isRequestTimeoutResponse(HttpResponse httpResponse) {
                "close".equalsIgnoreCase(httpResponse.headers().get(HttpHeaderNames.CONNECTION));
     }
 
+    /**
+     * Copies header value pairs of the specified {@linkplain HttpHeaders Armeria headers} to the
+     * {@link BiConsumer} excluding HTTP/2 pseudo headers that starts with ':'. This also converts
+     * {@link HttpHeaderNames#AUTHORITY} header to {@link HttpHeaderNames#HOST} header if
+     * the {@linkplain HttpHeaders Armeria headers} does not have one.
```

Comment:
I think you meant the opposite

```suggestion
     * the {@linkplain HttpHeaders Armeria headers} has one.
```

Reply:
I actually meant it. 😉 Let me change the word one to `{@link HttpHeaderNames#HOST}` because the original sentence is unclear. 😅

Reply:
Ah I see what you meant 😅